src/main/org/nlogo/prim/_breedsingular.scala

```diff
@@ -0,0 +1,29 @@
+// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo
+
+package org.nlogo.prim
+
+import org.nlogo.agent.{ AgentSet, Turtle }
+import org.nlogo.api.{ LogoException, Nobody$, Syntax }
+import org.nlogo.nvm.{ Context, EngineException, Reporter }
+
+class _breedsingular(breedName: String) extends Reporter {
+  override def syntax: Syntax =
+    Syntax.reporterSyntax(Array[Int](Syntax.NumberType), Syntax.TurtleType | Syntax.NobodyType)
+
+  override def toString: String = s"${super.toString}:$breedName"
+
+  override def report(context: Context) = report_1(context, argEvalDoubleValue(context, 0))
+
+  def report_1(context: Context, idDouble: Double): AnyRef = {
+    val id = validLong(idDouble)
+    if (id != idDouble)
+      throw new EngineException(context, this, s"$idDouble is not an integer")
+    val turtle = world.getTurtle(id)
+    if (turtle == null) return Nobody$.MODULE$
```

Comment:
Change `Nobody$.MODULE$` => `Nobody` and `import ... Nobody$` => `import .. Nobody`. Scala `object`s are referenced as `ObjectName$.MODULE$` in java, but just as `ObjectName` in scala. I'm not sure why it fails on travis but not on your machine, but perhaps performing `sbt clean` would cause it to fail on your machine.